src/main/java/com/example/onlinebookstore/config/MapperConfig.java

```diff
@@ -0,0 +1,13 @@
+package com.example.onlinebookstore.config;
+
+import org.mapstruct.InjectionStrategy;
+import org.mapstruct.NullValueCheckStrategy;
+
+@org.mapstruct.MapperConfig(
+        componentModel = "spring",
+        injectionStrategy = InjectionStrategy.CONSTRUCTOR,
+        nullValueCheckStrategy = NullValueCheckStrategy.ALWAYS,
+        implementationPackage = "<PACKAGE_NAME>.impl"
+)
```

Comment:
You can use mapper config directly in BookMapper with @Mapper annotation

Reply:
I didn't quite understand what you wrote about. What exactly to take and where to insert @Mapper

Reply:
No problem) 
I meant, that config 
        componentModel = "spring",
        injectionStrategy = InjectionStrategy.CONSTRUCTOR,
        nullValueCheckStrategy = NullValueCheckStrategy.ALWAYS,
        implementationPackage = "<PACKAGE_NAME>.impl"
        
could be used directly in BookMapper class in Mapper annotation. 

It's optional, you can leave as it is.